src/io/flutter/run/daemon/FlutterAppManager.java

```diff
@@ -324,30 +324,31 @@ private void addPendingCmd(int id, @NotNull Command command) {
   private Event eventHandler(@NotNull String eventName, @Nullable String json) {
     switch (eventName) {
       case "device.added":
-        return new DeviceAdded();
+        return new DeviceAddedEvent();
+      case "device.removed":
+        return new DeviceRemovedEvent();
       case "app.start":
-        return new AppStarted();
+        return new AppStartEvent();
+      case "app.started":
+        return new AppStartedEvent();
       case "app.debugPort":
-        return new AppDebugPort();
+        return new AppDebugPortEvent();
       case "app.log":
-        return new AppLog();
-      case "daemon.logMessage":
-        return new LogMessage();
+        return new AppLogEvent();
       case "app.stop":
```

Comment:
As part of https://github.com/flutter/flutter/pull/6679, "app.stop" --> "app.stopped"